### DISCUSSION TITLE
Progressively load playlists

### DISCUSSION THREAD

**Discussion by xxxserxxx** [spezifisch/stmps#30] [Ideas]
This relates to #21.

Large playlists can take a while to load, delaying display of the UI until all playlists are loaded. I want to background this process, which should be easy enough. The discussion I'm requesting is around if and how we show non-blocking progress to the user.  Do we implement a progress bar on the Playlist page? Add a new % complete in the increasingly crowded status bar?

As I mention in the ticket, I think this deserves a little up-front thought, because I can forsee needing similar progress status in other areas of the application -- for example, #26, which -- depending on the server DB size and performance, could take a few seconds. Whatever we do here should be reusable.

**Comment by xxxserxxx** [comment]
PR #32 submitted to address this.

**Comment by spezifisch** [comment]
Was merged.
